src/processes/follow-topic-process.js

```diff
@@ -92,9 +94,49 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   }
   // }
 };
+
+const sendSignedMessage = async ({
+  serverChannel,
+  targetUserId,
+  senderUsername,
+  communityMessageFormatUserId,
+  text
+}) => {
+  const message = await serverChannel?.sendMessage(
+    {
+      text,
+      user_id: communityMessageFormatUserId,
+      is_auto_message: true
+    },
+    {skip_push: true}
+  );
+
+  const notificationPayload = {
+    title: senderUsername,
+    body: `${text?.substring(0, 100)}`
+  };
+
+  const dataPayload = {
+    type: 'message.new',
+    channel_id: serverChannel?.id,
+    message: text,
+    message_schema: 'text',
+    attachment: '',
+    created_at: message?.message?.created_at || new Date().toISOString(),
+    is_annoymous: 'false',
+    priority: 'high',
+    content_available: 'true'
+  };
+
+  sendChatNotificationByChannelMembers([targetUserId], {
+    notification: notificationPayload,
+    data: dataPayload
+  });
+};
+
 const followTopicProcess = async (job, done) => {
   try {
-    console.info(`running job register process ! with id ${job.id}`);
+    console.info(`running job follow topic process ! with id ${job.id}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The console log message has been updated to reflect the correct job name. This is a good change for clarity and debugging purposes.
```diff
- console.info(`running job register process ! with id ${job.id}`);
+ console.info(`running job follow topic process ! with id ${job.id}`);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->